src/daemon/daemon_config.cpp

```diff
@@ -104,10 +104,22 @@ std::unique_ptr<const mp::DaemonConfig> mp::DaemonConfigBuilder::build()
     auto multiplexing_logger = std::make_shared<mpl::MultiplexingLogger>(std::move(logger));
     mpl::set_logger(multiplexing_logger);
 
+    auto storage_path = qgetenv("MULTIPASS_STORAGE");
+
     if (cache_directory.isEmpty())
-        cache_directory = MP_STDPATHS.writableLocation(StandardPaths::CacheLocation);
+    {
+        if (!storage_path.isEmpty())
+            cache_directory = storage_path + "/data";
```

Comment:
Better use `QDir::filePath`, to avoid the separator.

Reply:
Was trying to avoid the `QString` → `QDir` → `QString` dance… If `multipass::path` was actually something smarter…